README.md

```diff
@@ -47,35 +45,47 @@ Workers are built in their own bin directories:
+Run `./clean.sh` to delete all build files, including worker binaries and any intermediate files generated during the build process. Alternatively, run `git clean -xdf`.
 
-## Running the project
+## Running the project locally
 
-To launch a local instance of SpatialOS running the project,  run 
+To launch a local instance of SpatialOS running the project, run
 ```bash
-$ spatial alpha local launch --launch_config ./deployment.json
-``` 
+$ spatial alpha local launch
+```
 from the SpatialOS directory (or from any location by adding the `--main_config=\<path to spatialos.json\>` flag). This starts SpatialOS locally and runs the server workers `HelloWorker` and `DiceWorker`.
 
-Now you can connect game clients. You can find the client binaries in `OtherWorkers/Interactive/client/bin/x64/ReleaseWindows` (or `ReleaseMacOS` for Mac).
+You can optionally set the `--launch_config` flag to specify a filepath to the [launch configuration](docs/reference/launch-configuration.md) of your deployment. If the flag is not set, the spatial CLI will use launch configuration specified in the `launch_config` field of your [project configuration](docs/reference/project-configuration.md) as a fall back.
+
+As soon as your deployment is running, you can connect game clients to it. You can find the client binaries in `OtherWorkers/Interactive/client/bin/x64/ReleaseWindows` (or `ReleaseMacOS` for Mac).
 Connect your client by opening a second terminal to run the binary directly (from inside the `ReleaseWindows` or `ReleaseMacOS` directories):
 * Windows: `./Client.exe localhost 7777 <client_id>`
 * macOS: `mono --arch=64 Client.exe localhost 7777 <client_id>`
 
 This connects a client that pings the `HelloWorker` and `DiceWorker` every few seconds and then prints the response.
 
-## Logs location
+### Logs location
 The logs are stored in `logs` subdirectory of the location where your `spatialos.json` file is. For example, if my `spatialos.json` file is in `/workspace/project/SpatialOS`, the SpatialOS logs will go to `/workspace/project/SpatialOS/logs`.
 
+## Running the project in the cloud
+
+Upload your project assembly to the cloud:
+```bash
+$ spatial alpha cloud upload -a <your-assembly-name>
+```
+
+Start a deployment in the cloud
+```bash
+$ spatial alpha cloud launch -d <your-deployment-name> -a <your-assembly-name>
+```
+
 ## Known Issues
-* Our [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher) currently only supports Unity and Unreal based [client-workers](docs/reference/client-worker-configuration.md). As a result, if you deploy this project to the cloud, you can't start the client-worker provided in this project from the Launcher.
+* (There are no known issues at the moment.)
 
 ## Reference documentation
-[Main project configuration](docs/reference/project-configuration.md)
-
-[World configuration](docs/reference/world-configuration.md)
+[Project configuration](docs/reference/project-configuration.md)
```

Comment:
How would you feel about adding "file" to the end of all of these? I think I'm naturally wanting to do that because of the potential for confusion (with the current project layout) between the launch configuration _file_ and the launch configuration _section_ of the worker configuration _file_. I know in the FPL there are certain similar things (e.g. there's a launchConfig _field_ inside one of the files). 

Do you reckon it's worth adding "file" wherever we're talking about a file? Or is that overkill in your opinion?

Reply:
Let's just add `file` across the board. Screw it.